src/fiatExchanges/PaymentMethodSection.tsx

```diff
@@ -0,0 +1,245 @@
+
+  const toggleExpanded = () => {
+    LayoutAnimation.easeInEaseOut()
+    setExpanded(!expanded)
+  }
+  if (!sectionQuotes.length) {
+    setNoPaymentMethods(true)
+    return null
+  }
+  const renderFeeAmount = (quote: SimplexQuote | ProviderQuote, postFix: string) => {
+    const feeAmount = getFeeValueFromQuotes(quote)
+
+    if (feeAmount === undefined) {
+      return null
+    }
+
+    return (
+      <Text>
+        <CurrencyDisplay
+          amount={{
+            value: 0,
+            localAmount: {
+              value: feeAmount,
+              currencyCode: localCurrency,
+              exchangeRate: 1,
+            },
+            currencyCode: localCurrency,
+          }}
+          showLocalAmount={true}
+          hideSign={true}
+          style={styles.fee}
+        />{' '}
+        {postFix}
+      </Text>
+    )
+  }
+  return (
+    <View style={styles.container}>
+      <Touchable
+        onPress={
+          isExpandable
+            ? toggleExpanded
+            : ((quoteOnPress(sectionQuotes[0]) as unknown) as () => void)
+        }
+      >
+        <View>
+          <Expandable
+            arrowColor={colors.greenUI}
+            containerStyle={{
+              ...styles.expandableContainer,
+              paddingVertical: isExpandable ? (expanded ? 22 : 27) : 16,
+            }}
+            isExpandable={isExpandable}
+            isExpanded={expanded}
+          >
+            {isExpandable ? (
+              <>
+                <View style={styles.left}>
+                  <Text style={styles.category}>
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.card')
+                      : t('selectProviderScreen.bank')}
+                  </Text>
+                  {!expanded && (
+                    <Text style={styles.fee}>
+                      {renderFeeAmount(sectionQuotes[0].quote, t('selectProviderScreen.minFee'))}
+                    </Text>
+                  )}
+                </View>
+
+                <View style={styles.right}>
+                  <Text style={styles.providerDropdown}>
+                    {t('selectProviderScreen.numProviders', { count: sectionQuotes.length })}
+                  </Text>
+                </View>
+              </>
+            ) : (
+              <>
+                <View style={styles.left}>
+                  <Text style={styles.category}>
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.card')
+                      : t('selectProviderScreen.bank')}
+                  </Text>
+                  <Text style={styles.fee}>
+                    {renderFeeAmount(sectionQuotes[0].quote, t('selectProviderScreen.fee'))}
+                  </Text>
+                  <Text style={styles.topInfo}>
+                    {t('selectProviderScreen.idRequired')} |{' '}
+                    {paymentMethod === PaymentMethod.Card
```

Comment:
This assumes that all card transfers will settle in < 1 hour, and all bank transfers will take 1-3 days, right? The  behavior here is sort of opaque, but I imagine that this will have to be generalized/refactored anyhow once FiatConnect providers are supported (in particular, since we'll be receiving this sort of information dynamically).